src/compiler/scala/tools/nsc/util/package.scala

```diff
@@ -116,17 +116,6 @@ package object util {
 
   lazy val trace = new SimpleTracer(System.out)
 
-  // These four deprecated since 2.10.0 are still used in (at least)
-  // the sbt 0.12.4 compiler interface.
-  @deprecated("Moved to scala.reflect.internal.util.Position", "2.10.0")
-  type Position = scala.reflect.internal.util.Position
```

Comment:
Removing `Position` may prevent the `compile` task in sbt 0.13 - https://github.com/sbt/sbt-zero-thirteen/blob/v0.13.17/compile/interface/src/main/scala/xsbt/DelegatingReporter.scala#L18

Reply:
Ok, I'll put back and update the version in the comment to sbt 0.13.

Reply:
Or possibly 2.13 is a good moment to drop support for 0.13.

Reply:
> Or possibly 2.13 is a good moment to drop support for 0.13

I wish it were, but I don't think so

Reply:
Agree. Could be something to aim for in 2.14, though.

Reply:
Maybe 0.13 could drop support for 2.13 instead.

Reply:
Or release 0.14 as an EOL release: It doesn't support any Scala version.  End of line.

Reply:
Or release 0.14 as an EOL release: It doesn't support any Scala version.  End of line.

Reply:
@ashawley should definitely write the next April 1 release notes.

Reply:
> I wish it were, but I don't think so

Given sbt 0.13 completed its EOL before Scala 2.13 was released, I think it's fair to change things in Scala 2.13 even if it breaks sbt 0.13.  That said, I don't think this change is significant enough to start that process.

Reply:
I'd advocating for keeping sbt 0.13 support in part for the interest of Scala 2.13. As an sbt maintainer, I am happy to say "just use latest sbt 1 patch." But assuming some percentage (source needed) of the OSS and enterprise world builds are on sbt 0.13, _not_ having sbt 0.13 would be an additional hurdle to Scala 2.13 adoption.

Reply:
OTOH, having a high bar keeps them from complaining about the other hurdles to adoption.

Reply:
Just so you know: Scala.js uses sbt 0.13.17 ... ScalaTest 3.0.x as well.